### DISCUSSION TITLE
Bluetooth Compatibilty?

### DISCUSSION THREAD

**Discussion by jimbo601** [justLV/onju-voice#46] [Ideas]
One of my biggest gripes about the google home mini vs the amazon alexa is that (from what I've been told) the google home mini doesn't have bluetooth compatibility to act as a bluetooth speaker. Would it be possible to add a bluetooth chip to the board to make it both bluetooth and wifi compatible? Apologies if I'm being redundant with something that's already been covered lol

**Comment by jimbo601** [comment]
Additionally, how hard would it be to integrate Zigbee/Z-wave/Matter connectivity into the board? I'm trying to think of how best to make the devices future-proof so the investment is worth it for people haha.

**Comment by justLV** [comment]
The ESP32-S3 does support BT5.0 (classic + BLE) so it should be possible in software.

Try something like the `ESP32-a2dp` library.

I haven't tried this before so a disclaimer is you'll need to know what you're doing ;)
